app/models/guest_users_detail.rb

```diff
@@ -1,2 +1,4 @@
 class GuestUsersDetail < ApplicationRecord
```

Comment:
Why is this called Details?  Should it be called something more like membership or record?